docs/spec/spec.md

```diff
@@ -1,104 +1,414 @@
-# Specification: Ballerina Soap Library
+# Specification: Ballerina SOAP Library
 
 _Owners_: @shafreenAnfar @MadhukaHarith92  
 _Reviewers_: @shafreenAnfar  
 _Created_: 2023/06/07  
-_Updated_: 2023/08/14  
+_Updated_: 2023/12/01
 _Edition_: Swan Lake
 
 ## Introduction
-This is the specification for the Soap standard library of [Ballerina language](https://ballerina.io/), which provides APIs to send an ordinary XML request to a SOAP backend by specifying the necessary details to construct a SOAP envelope.
 
-The Soap library specification has evolved and may continue to evolve in the future. The released versions of the specification can be found under the relevant GitHub tag.
+This is the specification for the SOAP standard library of [Ballerina language](https://ballerina.io/), which provides APIs to send an ordinary XML request to a SOAP backend by specifying the necessary details to construct a SOAP envelope.
 
-If you have any feedback or suggestions about the library, start a discussion via a [GitHub issue](https://github.com/ballerina-platform/ballerina-standard-library/issues) or in the [Discord server](https://discord.gg/ballerinalang). Based on the outcome of the discussion, the specification and implementation can be updated. Community feedback is always welcome. Any accepted proposal, which affects the specification is stored under `/docs/proposals`. Proposals under discussion can be found with the label `type/proposal` in GitHub.
+The SOAP library specification has evolved and may continue to evolve in the future. The released versions of the specification can be found under the relevant GitHub tag.
```

Comment:
```suggestion
The SOAP package specification has evolved and may continue to evolve in the future. The released versions of the specification can be found under the relevant GitHub tag.
```

Reply:
Fixed in [0582c3e](https://github.com/ballerina-platform/module-ballerina-soap/pull/117/commits/0582c3eb0e040ac6129a04c88fb6b873ee97c791)